Makefile

```diff
@@ -32,7 +32,7 @@ clean:
 
 .PHONY: candump
 candump:
-	@pipenv run python -m tool.script.read_bus $(port)
+	@pipenv run python -m tool.script.read_bus $(dev) $(id_rx) $(id_tx)
```

Comment:
I'd prefer to have the read_bus being able to read via TCP, it's useful for me to debug remotely, and we don't really care about 1s freeze in debug mode do we?

Reply:
We have to test on the pi more extensively to check that those billions of isotpservers dont generate any issue, I would rather take them out first, then verify this.

Though, even if they cause issues, we could start them for debug purposes once in a while.

Reply:
fine, wateva